docs/ui/auth/fragments/web/authenticator.md

```diff
@@ -0,0 +1,1251 @@
+<amplify-authenticator></amplify-authenticator>
+
+## Installation
+
+<docs-filter framework="react">
```

Comment:
NIT/Non-Blocking: I believe as a follow up to this we can see about splitting the code snippets up into separate fragments. I feel it'll make it cleaner in the long run for maintainability